internal/web/server.go

```diff
@@ -0,0 +1,18 @@
+//go:build web
+
+package web
+
+import (
+	"github.com/weka/gohomecli/internal/web/api"
```

Comment:
minor: please, sort imports with goimports

Reply:
forgot to setup pre-commit locally, good point